src/lib/Preview.js

```diff
@@ -486,6 +486,11 @@ class Preview extends EventEmitter {
         // Init performance logging
         this.logger = new Logger(this.location.locale);
 
+        // Save reference to the file when a well-formed file object is present in options
+        if (this.previewOptions.file) {
+            this.updateFileCache(this.previewOptions.file);
+        }
```

Comment:
Remove this. I'll try and get a different PR in this week for allowing the file object to be directly sent into preview.show() as the first param.

The signature will be like:

```
    /**
     * Primary function for showing a preview of a file.
     *
     * @param {string|Object} fileIdOrFile - Box File ID or well-formed Box File object
     * @param {string|Function} token - Access token string or generator function
     * @param {Object} [options] - Optional preview options
     * @return {void}
     */
```